src/app/game-state/followers.service.ts

```diff
@@ -273,12 +277,13 @@ export class FollowersService {
       this.logService.addLogMessage("A new follower shows up, but they were a " + job + " and you don't want any more of those.","STANDARD","FOLLOWER");
       return;
     }
-
-    this.logService.addLogMessage("A new follower has come to learn at your feet.","STANDARD","FOLLOWER");
+    
+    const lifespanDivider = this.followerLifespanDoubled ? 5 : 10;
```

Comment:
Added in follower lifespan doubling as an achievement reward for getting a follower to level 100. If a player can't do that then they can't continue with swimming. It's an intentional wall.